### PR TITLE
add AdminTitleBar to manage space component

### PR DESCRIPTION
**Pull request by denisdimitrov**
Adds AdminTitleBar component with the buttons and texts from the IA design.

Fixes: https://github.com/sparkletown/internal-sparkle-issues/issues/1486

### CODE REVIEW

**Review by goran-peoski-work — CHANGES_REQUESTED** [review]
There is a way to solve this without injecting class names into `AdminTitleBar`. It is meant to be a single child (in the page, not just its parent component) so it can be safely targeted with `.AdminTitleBar` 
- https://github.com/sparkletown/sparkle/pull/2610

File: src/components/organisms/AdminVenueView/AdminVenueView.tsx [hunk 1]
```diff
@@ -115,6 +130,26 @@ export const AdminVenueView: React.FC = () => {
...
     <WithNavigationBar withSchedule>
       <AdminRestricted>
+        <AdminTitleBar className="AdminVenueView__title-bar">
+          <ButtonNG onClick={navigateToHome} iconName={faArrowLeft}>
+            Back to Dashboard
+          </ButtonNG>
+          <AdminTitle>Edit {space.name}</AdminTitle>
+          <div>
+            <ButtonNG variant="danger" onClick={showDeleteModal}>
+              Delete space
```
Comment (alexey-petrov) [inline src/components/organisms/AdminVenueView/AdminVenueView.tsx:140]: let's use `SPACE_TAXON.lower`

File: src/components/organisms/AdminVenueView/AdminVenueView.tsx [hunk 2]
```diff
@@ -115,6 +130,26 @@ export const AdminVenueView: React.FC = () => {
...
+            </ButtonNG>
+            <ButtonNG
+              isLink
+              newTab
+              linkTo={spaceSlug ? venueInsideUrl(spaceSlug) : undefined}
+              variant="primary"
+            >
+              Visit Space
```
Comment (alexey-petrov) [inline src/components/organisms/AdminVenueView/AdminVenueView.tsx:148]: `SPACE_TAXON.capital`

**Review by alexey-petrov — CHANGES_REQUESTED** [review]
A few strings to replace with constants

**Review by goran-peoski-work — APPROVED** [review]
Approve with expectation to resolve some issues through sub-PR merge to staging.

File: src/components/organisms/AdminVenueView/AdminVenueView.tsx (hunk 2 above)
Comment (denisdimitrov) [reply src/components/organisms/AdminVenueView/AdminVenueView.tsx]: Fixed

File: src/components/organisms/AdminVenueView/AdminVenueView.tsx (hunk 1 above)
Comment (denisdimitrov) [reply src/components/organisms/AdminVenueView/AdminVenueView.tsx]: Fixed.

**Review by alexey-petrov — APPROVED** [review]
LGTM

**Pull request merged**